### DISCUSSION TITLE
Segmentation

### DISCUSSION THREAD

**Discussion by Donalcon** [obss/sahi#955] [Q&A] [answered]
Hi Guys,

Any updates on masks?

No model seems to return masks? I've seen many people posing questions about segmentation.

**Comment by fcakyon** [comment, marked as answer]
Segmentation mask support for yolov8, mmdetection, detectron2, torchvision models is live with the latest release! @Donalcon @qgeissmann @FilipeMendesVMD @lapers  https://github.com/obss/sahi/discussions/1051

There is also 5x memory and speed improvement when working with masks!

Check the demo notebook on how it works: https://github.com/obss/sahi/blob/main/demo/inference_for_yolov8.ipynb
